string rosbag
uint64 size
float64 time
